BUILD

```diff
@@ -26,8 +26,8 @@ load("@vaticle_dependencies//distribution:deployment.bzl", "deployment")
 load("@vaticle_dependencies//distribution/artifact:rules.bzl", "artifact_repackage")
 load("@vaticle_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
 load("@vaticle_dependencies//tool/release/deps:rules.bzl", "release_validate_deps")
-load("@io_bazel_rules_docker//container:bundle.bzl", "container_bundle")
-load("@io_bazel_rules_docker//container:image.bzl", "container_image")
+load("@io_bazel_rules_docker//container:bundle.bzl", docker_container_bundle = "container_bundle")
+load("@io_bazel_rules_docker//container:image.bzl", docker_container_image = "container_image")
```

Comment:
In the scope of TypeDB's root `BUILD` file, the rule alias `docker_container_image` is clearer than the source symbol `container_image`.